docs/src/api/json_endpoints.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: JSON Endpoints
+---
+
+Lookbook currently implements a single JSON endpoint, reachable by adding the JSON format to the root Lookbook URL. It returns previews and their examples:
```

Comment:
You ought to change the wording here to reflect the new route.

Reply:
Done!